docusaurus.config.js

```diff
@@ -1,3 +1,4 @@
+const postcssPreset = require('postcss-preset-env');
```

Comment:
don't we need to install this? 

(my favorite review card: undeclared dependency)